src/scripts/domPopulator.js

```diff
@@ -0,0 +1,52 @@
+import rainGif from '../assets/images/rain.gif';
+import clearPng from '../assets/images/clear-sky.png';
+import cloudsGif from '../assets/images/clouds.gif';
+import snowGif from '../assets/images/snow.gif';
+
+function setBackgroundImage(status) {
+  let url = '';
+  switch (status) {
+    case 'Rain':
+      url = rainGif;
+      break;
+    case 'Thunderstorm':
+      url = rainGif;
+      break;
+    case 'Drizzle':
+      url = rainGif;
+      break;
+    case 'Clear':
+      url = clearPng;
+      break;
+    case 'Snow':
+      url = snowGif;
+      break;
+    default:
+      url = cloudsGif;
+      break;
+  }
+  return url;
+}
+
+export default function domPopulator(object, metricValue = 'Metric') {
```

Comment:
- [ ] Let's use ES6 more! Use arrow functions instead of the keyword 'function'.
[Check this article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) on more about arrow functions and how to use them.